tox.ini

```
@@ -37,4 +37,4 @@ deps =
     mypy==0.780
 commands =
     mypy --show-error-codes --ignore-missing-imports \
-        --follow-imports=skip {posargs: xtractmime setup.py tests}
+        --follow-imports=skip {posargs: xtractmime setup.py tests}
```

```suggestion
        --follow-imports=skip {posargs: xtractmime setup.py tests}
        
```
This is a no-change, the suggestion will remove it from the PR.